### DISCUSSION TITLE
REVISIONES PREVIAS

### DISCUSSION THREAD

**Discussion by carlos319** [rramosp/ai4eng.v1#290] [00 - General]
Muy buenas noches @Felipe-RA  queremos realizar una revision previa a la entrega, puedes de pronto a las 6pm mañana sábado 21 de octubre, sino tambien me avisas y gracias!

**Comment by Felipe-RA** [comment]
Hola @carlos319 

Apenas pude echarle un vistazo a esto. 

Se que es algo tarde, pero si estas libre y ves esto a tiempo escribeme por interno al correo institucional
